srcpkgs/openttd/template

```diff
@@ -15,15 +14,14 @@ maintainer="Jürgen Buchmüller <pullmoll@t-online.de>"
 license="GPL-2 Custom"
 homepage="https://www.openttd.org/"
 distfiles="
- http://binaries.openttd.org/releases/${version}/openttd-${version}-source.tar.xz
- http://bundles.openttdcoop.org/opengfx/releases/${_gfxver}/opengfx-${_gfxver}.zip
- http://bundles.openttdcoop.org/opensfx/releases/${_sfxver}/opensfx-${_sfxver}.zip
-"
-checksum="
- c2d32d9d736d27202a020027a3729ae763f5432ae6f424891e57a4095eeb087f
- 3d136d776906dbe8b5df1434cb9a68d1249511a3c4cfaca55cc24cc0028ae078
- 3574745ac0c138bae53b56972591db8d778ad9faffd51deae37a48a563e71662
+ https://proxy.binaries.openttd.org/openttd-releases/${version}/openttd-${version}-source.tar.xz
```

Comment:
I think the Void style policy is to not put the first item on a separate line, just as `checksum="` has the first item immediately after the `"`, so please remove the line break after `ditfiles="`.